source/outlines/2016-03-07.markdown

```diff
@@ -0,0 +1,127 @@
+---
+title: 20160307
+layout: outline
+---
+
+## All
+
+### Word of the Day:
+
+* tenet - noun - any opinion, principle, dogma, belief, or doctrine that a
+person holds or maintains as true.
+
+
+## 1508
+
+### Dangerous Denver Kick Off (9:00 - 10:00)
+
+### Real Time Evaluations (10:00 - 12:00)
+
+#### With Meeka
+* 10:00 - Tyler Komoroske
+* 10:30 - Aaron Careaga
+* 11:00 - Matt Stjernholm
+* 11:30 - Jill Donohue
+
+### With Jorge
+* 10:00 - Robbie Lane
+* 10:30 - Matt Rooney
+* 11:00 - Sekhar Paladugu
+* 11:30 - Emily Dowdle
+
+
+### Real Time Evaluations (1:00 - 4:00)
+
+#### With Meeka
+* 1:00 - Ryan Johnson
+* 1:30 - Adam Jensen
+* 2:00 - Jason Pilz
+* 2:30 - Ross Edfort
+* 3:00 - Michael Wong
+* 3:30 - Jessica Goulding
+
+#### With Tess
+* 1:00 - Marlo Major
+* 1:30 - Pat Wey
+* 2:00 - Jhun de Andres
+* 2:30 - Amber Crawford
+* 3:00 - Torie Warren
+
+
+## 1510
+
+### Getting Started With Workers (9:00 - 12:00)
+
+Join Josh in Classroom C.
+
+### Project Work Time (1:00 - 4:00)
+
+Work
+
+
+## 1511
+
+### Introduction to JavaScript (9:00 - 12:00)
+
+Join Andrew in Classroom C.
+
+### Little Shop Work Time and Checkins (1:00 - 4:00)
+
+Schedule TBD.
+
+
+## 1602
+
+### Higher Order Ruby: Writing Methods that Accept Blocks (9:00 - 10:30)
+
+Classroom A.
+
+### Final Assessments (10:30 - 12:00)
```

Comment:
I want this to be consistent with the star wars theme, but not sure what "A New Hope" communicates about assessments. :P